app/ide-desktop/lib/dashboard/src/tailwind.css

```diff
@@ -8,6 +8,16 @@ body {
   background-blend-mode: lighten;
 }
 
+.Toastify__toast {
+  border-radius: 14px;
```

Comment:
if we want to follow the style, we probably want to use tailwind classes instead?

so something like `toastClassName="text-sm leading-170 bg-frame-selected rounded-2xl backdrop-blur-3xl"` on the `toastify.ToastContainer`. not 100% sure `toastClassName` is the right `className`, should probably test it just to make sure

Reply:
Works as advertised, thank you!